### ISSUE TITLE
New repository for builder updates (?)

### ISSUE THREAD

**Issue by bensheldon** [jimweirich/builder#56]
A new version of builder, 3.2.3, was published to [Rubygems on Jan 13, 2017](https://rubygems.org/gems/builder/versions/3.2.3) from the @tenderlove's fork: https://github.com/tenderlove/builder/commit/4566addca4ec766db7e8a9dac758d0b47c85fda0

Just noting to save anyone a few minutes of sleuthing.

**Comment by junaruga** [comment]
I want someone to change "Source Code" link to @tenderlove's repo, if it is now main repo.
https://rubygems.org/gems/builder
I want to contribute. but I am not sure which is a primary repo.

**Comment by watt** [comment]
@tenderlove is your fork now the official source repo for builder?

**Comment by junaruga** [comment]
@tenderlove ping ping :)

**Comment by tenderlove** [comment]
Yes my fork is official. The link will get updated next time I push.

**Comment by junaruga** [comment]
@tenderlove ok great news! Thanks for the maintaining it.

**Comment by junaruga** [comment]
@tenderlove is it possible to active "Issues" in your repository: https://github.com/tenderlove/builder ?

**Comment by lloeki** [comment]
@tenderlove what do you think about @junaruga's suggestion?

**Comment by svoop** [comment]
@tenderlove Just a little heads-up since two years have passed: In order to prevent confusions, thanks for considering activating issues on tenderlove/builder as well as to update the links (most notable the link to source) on [Rubygems](https://rubygems.org/gems/builder). Thanks a bunch!

**Comment by ShockwaveNN** [comment]
@tenderlove maybe you should activate issues for https://github.com/tenderlove/builder?
I wanted to report that there is no changelog entry for 3.2.4
